### PR TITLE
19 create button base component

### PR DESCRIPTION
**Pull request by Vadis00**
Button example:

```
-   <app-button text="Sign In" class="btn"></app-button>
-   <app-button text="Sign In" class="btn disabled"></app-button>
-   <app-button text="Create Account" class="btn mini"></app-button>
-   <app-button text="Create Account" class="btn mini disabled"></app-button>
-   <app-button text="Create Account" class="btn stroke"></app-button>
-   <app-button text="Create Account" class="btn stroke disabled"></app-button>
-   <app-button text="Create Account" class="btn fill"></app-button>
-   <app-button text="Create Account" class="btn fill disabled"></app-button>
```

### CODE REVIEW

File: frontend/src/app/component/button/button.component.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<button
+	(click)="onClick()"
+	class="{{ buttonClass }}">
+	<span>{{ buttonText }}</span>
+</button>
```
Comment (FeloniukM) [inline frontend/src/app/component/button/button.component.html]: Must be white space

File: frontend/src/app/component/button/button.component.sass [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    background: #EE2A64
+    border-radius: 50px
+    &:hover
+        background: #E30547
+        color: white
+    &.disabled
+        color: white
+        background: none
+        background-color: #CBCDD3
+        cursor: not-allowed
```
Comment (FeloniukM) [inline frontend/src/app/component/button/button.component.sass]: same

File: frontend/src/app/component/button/button.component.html (hunk 1 above)
Comment (Vadis00) [reply frontend/src/app/component/button/button.component.html]: done

**Review by FeloniukM — DISMISSED** [review]

**Review by ismv — CHANGES_REQUESTED** [review]
Please pull the dev updates and use color scheme for all the colors you added

**Review by ismv — APPROVED** [review]

**Review by FeloniukM — APPROVED** [review]

**Pull request merged**